modules/KalturaSupport/components/dualScreen/dualScreen.js

```diff
@@ -308,10 +323,12 @@
 
 					//Attach the primaryPlayerContainer to the primary display
 					var primaryDisplay = this.displays.getPrimary();
+					primaryPlayerContainer.attr('data-display-type','video');
 					primaryDisplay.attachView(primaryPlayerContainer);
 
 					//Attach the secondaryDisplay to the second display
 					var secondaryDisplay = this.displays.getSecondary();
+					secondaryPlayerContainer.attr('data-display-type','presentation');
```

Comment:
@OrenMe agreed, this is why I'm not using that metadata in the final implementation. Will remove this also.